### PR TITLE
Add optional load balancer configurations

### PR DESCRIPTION
**Pull request by christian-roggia**
We deployed dgraph on GKE and would like to configure and internal load balancer in order to make available in the entire VPC. Unfortunately, right now there is no way to inject annotations or general load balancer configuration in any of the services provided by the Helm chart.

This PR enables the configuration of load balancer configs, service annotations, and makes it easy to disable the ratel deployment.

This PR addresses some of the features discussed in https://github.com/dgraph-io/charts/issues/6.

TODO list:

- [x] bump version
- [x] update README
- [x] update `values.yaml`
- [x] add load balancer configurations (`loadBalancerIP`, `loadBalancerSourceRanges`, and `externalTrafficPolicy`)
- [x] add annotations to services (`annotations`)
- [x] allow disabling ratel through a boolean value

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/charts/24)

### CODE REVIEW

**Comment by christian-roggia** [comment]
I just noticed this PR includes changes made in https://github.com/dgraph-io/charts/pull/20

File: charts/dgraph/templates/alpha-svc.yaml [hunk 1]
```diff
@@ -27,3 +37,7 @@ spec:
     chart: {{ template "dgraph.chart" . }}
     component: {{ .Values.alpha.name }}
     release: {{ .Release.Name }}
+  {{- with .Values.alpha.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```
Comment (MichelDiz) [inline charts/dgraph/templates/alpha-svc.yaml:43]: Can you add a new line here?

Comment (christian-roggia) [reply charts/dgraph/templates/alpha-svc.yaml]: done

**Comment by MichelDiz** [comment]
Pinging @danielmai, @darkn3rd  or @prashant-shahi to add one more review to this PR.

**Comment by darkn3rd** [comment]
We'll look at adding these before the next version is published.

**Comment by darkn3rd** [comment]
The aforementioned issue https://github.com/dgraph-io/charts/issues/6 is now at https://discuss.dgraph.io/t/feedback-ratel-should-be-disabled-by-default-and-available-on-port-80-443/8261

**Comment by darkn3rd** [comment]
I created a similar PR that is compatible with the current code based on this PR: 

* https://github.com/dgraph-io/charts/pull/52

**Pull request closed without merging**